Modify inventory summary FA, refs #10591

jraddaoui opened the pull request:
Change file/item level information:

- Remove access status and container columns from table.
- Add physical location column.
- Modify column widths and spanned columns.
- Include an extra row per file/item with the following EAD elements:
  - origination.
  - materialspec.
  - abstract.
  - note.
  - scopecontent.
  - userestrict (access status).
- Make title bold to differentiate it from the new extra content.

Other cosmetic changes:

- Improve borders declaration.
- Avoid "File / item list" header if there are no file/item children.
- Remove unnecessary `<xsl:if test="*">` condition.
- Add `type` label to elements like notes at the series level.
- Minor code clean-ups.

---

Part of this work comes from @mvail pull request (https://github.com/artefactual/atom/pull/471) and it was not included on https://github.com/artefactual/atom/pull/978.

mcantelon commented on lib/task/pdf/ead-pdf-inventory-summary.xsl:
```diff
@@ -1452,53 +1443,47 @@
                         <xsl:apply-templates select="ead:did" mode="dscSeriesTitle"/>
                         <xsl:apply-templates select="ead:did" mode="dscSeries"/>
                         <xsl:value-of select="self::bioghist"/>
                         <!--<xsl:apply-templates select="child::*[not(ead:did) and not(self::ead:did)]" mode="dsc"/>-->
```
Accidentally left in?

jraddaoui replied:
Already there but I'll remove it.

mcantelon reviewed (approved):
Looks good to me (other than one bit of commented out markup)!

jraddaoui committed:
Modify inventory summary FA, refs #10591

Change file/item level information:

- Remove access status and container columns from table.
- Add physical location column.
- Modify column widths and spanned columns.
- Include an extra row per file/item with the following EAD elements:
  - origination.
  - materialspec.
  - abstract.
  - note.
  - scopecontent.
  - userestrict (access status).
- Make title bold to differentiate it from the new extra content.

Other cosmetic changes:

- Improve borders declaration.
- Avoid "File / item list" header if there are no file/item children.
- Remove unnecessary `<xsl:if test="*">` condition.
- Add `type` label to elements like notes at the series level.
- Minor code clean-ups.

jraddaoui commented:
Thanks @mcantelon!
